merchant/src/catalog/service.rs

```diff
@@ -60,25 +60,25 @@ pub trait CatalogService: BulkDocumentConverter<Id = CatalogId<Self>> + Commande
     async fn bulk_create(
         &self,
         account: &Self::Account,
-        catalog: Vec<CatalogObjectBulkDocument<String>>, // we use string type for Id to enable referecing
+        catalog: &Vec<CatalogObjectBulkDocument<String>>, // we use string type for Id to enable referecing
```

Comment:
```suggestion
        catalog: &[CatalogObjectBulkDocument<String>], // we use string type for Id to enable referecing
```
Receiving a slice as parameter is often a better practice than passing the reference to a Vec as it makes the method more flexible(you can still pass the Vec but also sub-slices of it).